utils/conversions.ts

```diff
@@ -115,33 +127,61 @@ export const toApiUserBonus = (domainObj: UserBonus): BonusActivationItem => {
   };
 };
 
-function calculateMaxBonusAmount(
-  numberOfFamilyMembers: number
+/**
+ * Returns the maximum bonus amount in Euros from the total number of family
+ * members.
+ */
+function calculateMaxBonusAmountFromTotFamilyMembers(
+  numberOfFamilyMembers: FamilyMemberCount
 ): MaxBonusAmount {
-  return (numberOfFamilyMembers > 2
-    ? 500
-    : numberOfFamilyMembers === 2
-    ? 250
-    : numberOfFamilyMembers === 1
-    ? 150
-    : 0) as MaxBonusAmount;
```

Comment:
@BurnedMarshal @gunzip @balanza here we had an invalid cast from `0` to `MaxBonusAmount` (that must be within the range 150-501)